projects/components/src/time-picker/time-picker.component.ts

```diff
@@ -0,0 +1,69 @@
+import { ChangeDetectionStrategy, Component, ElementRef, EventEmitter, Input, Output, ViewChild } from '@angular/core';
+import { IconType } from '@hypertrace/assets-library';
+import { Time } from '@hypertrace/common';
+import { IconSize } from '../icon/icon-size';
+import { PredefinedTimeService } from '../time-range/predefined-time.service';
+
+@Component({
+  selector: 'ht-time-picker',
+  styleUrls: ['./time-picker.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `
+    <div class="time-picker">
+      <ht-popover class="time-selector" [closeOnClick]="true">
```

Comment:
Besides the values, how does this really differ from our select component? Could it just wrap that?

Reply:
+1

I think we can use the select component.

Reply:
It guess it could. But I don't want to do it now. have few other things to finish. This one is just extracted from date-time picker

Reply:
Please file a tech debt issue for this

Reply:
https://traceableai.atlassian.net/browse/ENG-6567

Reply:
Here you go